synapse/federation/replication.py

```diff
@@ -47,26 +45,6 @@ class ReplicationLayer(FederationClient, FederationServer):
     """
 
     def __init__(self, hs, transport_layer):
```

Comment:
can we kill off the `transport_layer` param (and hence get rid of this whole constructor)?

Reply:
meh in view of https://github.com/matrix-org/synapse/pull/2978/commits/ea7b3c4b1b829703a780418e6bb6b7860a5b5451 that's fine